packages/devtools_app/test/app_size/app_size_screen_test.dart

```diff
@@ -170,8 +170,10 @@ void main() {
       expect(breadcrumbs.length, 1);
       expect(breadcrumbs.first.text, equals('Root [6.0 MB]'));
       expect(find.byType(BreadcrumbNavigator), findsOneWidget);
-      expect(find.text('package:flutter'), findsOneWidget);
-      expect(find.text('dart:core'), findsOneWidget);
+      String text = 'package:flutter';
+      expect(textFinder(text), findsOneWidget);
```

Comment:
Once we change `getCaption` to return `null`, we should be able to search directly for the text with `find.richText(text)`, which should only return one widget (here and elsewhere).